client/js/campaign/campaignDetailComponent.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Client from '../api';
+import { Row, Col } from 'react-bootstrap';
+import { Table, Column, Cell } from 'fixed-data-table';
+import Moment from 'moment';
+import CampaignInfo from './campaignScorecard.jsx';
+import Promise from 'bluebird';
+
+class CampaignDetailComponent extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = { contributions: [] };
+    }
+
+    componentWillMount(props) {
+        const { id, candidateId } = this.props.params;
+        let that = this;
+        const detailsPromise = Client.getCampaignData(id);
+        const scorePromise = Client.getCandidate({id: candidateId, campaigns: [{campaignId: id}]});
+        Promise.join(detailsPromise, scorePromise, function(details, score){
+            that.setState({contributions: details, campaignScore: score[0].campaigns[0], candidate: score[0].candidate});
+        });
+    }
+
+    render() {
+        const { contributions, campaignScore, candidate } = this.state;
+        console.log(campaignScore);
+        if(contributions && campaignScore) {
+            return (
+                <Row>
+                    <Col xs={12}>
+                        <Row>
+                            <Col xs={12}>
+                                <h2>{candidate.name}</h2>
+                            </Col>
+                        </Row>
+                    </Col>
+                    <Col xs={3}>
+                        <CampaignInfo info={campaignScore}></CampaignInfo>
+                    </Col>
+                    <Col xs={9}>
+                        <Table rowsCount={contributions.length}
```

Comment:
What I can do is maybe fix the first three columns so they always show and the other ones are controlled by horizontal scroll bar.